src/lib/types/Message.ts

```diff
@@ -10,4 +10,5 @@ export type Message = Partial<Timestamps> & {
 	webSearchId?: WebSearch["_id"]; // legacy version
 	webSearch?: WebSearch;
 	score?: -1 | 0 | 1;
+	files?: string[]; // filenames
```

Comment:
nit: can it be renamed to `fileNames` ?

Reply:
I fixed the comment so it's more clear what's inside!